### ISSUE TITLE
E: Package 'firefox' has no installation candidate when running ./install.sh

### ISSUE THREAD

**Issue by wangk5** [mozilla/OpenWPM#611]
Hello,

I just tried to set up OpenWPM by running the install.sh script, but get this error:

```
Package firefox is not available, but is referred to by another package.
This may mean that the package is missing, has been obsoleted, or
is only available from another source

E: Package 'firefox' has no installation candidate
```

**Comment by wangk5** [comment]
Removing `firefox` seems to have resolve this issue. 
Also, removing line 45 https://github.com/mozilla/OpenWPM/blob/master/install-system.sh#L45 got the script to run successfully. 

Thanks!

**Comment by wangk5** [comment]
Although I was able to run install.sh, I now get the following errors when running demo.py

`libgtk-3.so.0: cannot open shared object file: No such file or directory
Couldn't load XPCOM.`
`RuntimeError: Firefox not found.  Did you run `./install.sh`?`

**Comment by englehardt** [comment]
What OS are you trying to install on?

Do you see a `firefox-bin` directory in the root directory of OpenWPM?

**Comment by birdsarah** [comment]
> `libgtk-3.so.0: cannot open shared object file: No such file or directory Couldn't load XPCOM.`

I expect that error in a docker container or server where there's no x server. In particular, I have a vague recollection that centos doesn't have libgtk-3 available.

Are you running in headless mode?

**Comment by birdsarah** [comment]
I believe this will be resolved with #648
